app/scripts/catalogue-sync.ts

```diff
@@ -77,9 +80,7 @@ async function syncDataCatalogue() {
   // convert to unix timestamp in ms
   const lastUpdate = lastUpdateData.last_update * 1000;
 
-  console.log(
-    `Last update: DB - ${previousUpdate}, API - ${lastUpdate}`,
-  );
+  console.log(`Last update: DB - ${previousUpdate}, API - ${lastUpdate}`);
   if (lastUpdate <= previousUpdate) {
     console.warn("Already on the newest data catalogue version, exiting.");
```

Comment:
Are we using this for logging also?

Reply:
Yes, otherwise the program would just terminate without letting you know why. So I think it's fine to keep this.